R/sumbetadiff.R

```diff
@@ -1,27 +1,36 @@
-##' Mode and Credible Interval Calculation for The Difference between Two Beta Distributions
+##' Mode and Credible Interval Calculation for The Difference between Two Beta Distributions.
 ##'
+##' @description `r lifecycle::badge("experimental")`
 ##'
-##' A function to summerize the characters of a betadiff distribution \code{\link{dbetadiff}}
+##' A function to summarize the characters of a betadiff distribution `[dbetadiff]`.
```

Comment:
```suggestion
##' A function to summarize the characters of a betadiff distribution [dbetadiff()].
```
(see https://roxygen2.r-lib.org/articles/rd-formatting.html#function-links as reference)

Reply:
thanks, I forgot that